src/app/components/Companies.js

```diff
@@ -1,31 +1,69 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import getCompaniesData from '../store/actions/getCompaniesData';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Chip from '@material-ui/core/Chip';
+import Link from '@material-ui/core/Link';
```

Comment:
Ahh sorry I wasn't clear, but it should be { Link } from 'react-router-dom'

Reply:
Ah right! I had noticed the JSON response and created a bug card last night. Thank you!